libzmix/src/signatures/ps/keys.rs

```diff
@@ -6,56 +6,43 @@ use super::{OtherGroup, SignatureGroup};
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Sigkey {
-    pub X: SignatureGroup,
+    pub x: FieldElement,
```

Comment:
This is a breaking change. Any code that depends on these won't work now. Please increment the version number of libzmix to 0.2.0